packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactRoot.java

```diff
@@ -28,6 +28,7 @@ public interface ReactRoot {
   @Nullable
   Bundle getAppProperties();
 
+  @Nullable
```

Comment:
We can remove this `@Nullable` annotation and all the new asserts on this diff

Reply:
updated. does the diff look correct now?